### PR TITLE
libobs: Remove unused variable in obs_source

### PR DESCRIPTION
**Pull request by norihiro**
### Description

Remove `last_audio_ts` in `struct obs_source`.

### Motivation and Context

The variable `last_audio_ts` was set but was never read.

### How Has This Been Tested?

OS: Fedora 38
Confirmed the build succeeded.

Also checked `git grep '\<last_audio_ts\>' libobs`.

### Types of changes

- Code cleanup (non-breaking change which makes code smaller or more readable)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by RytoEX** [comment]
It is not used for the async delay filter?
https://github.com/obsproject/obs-studio/blob/9c90c721a0b3134c00eb778d224b8ce97e1f1ec0/plugins/obs-filters/async-delay-filter.c#L202

**Comment by norihiro** [comment]
> It is not used for the async delay filter?

The async delay filter has the same variable name but it is declared inside the filter.

https://github.com/obsproject/obs-studio/blob/9c90c721a0b3134c00eb778d224b8ce97e1f1ec0/plugins/obs-filters/async-delay-filter.c#L34

`struct obs_source::last_audio_ts` is defined in obs-internal.h so that the variable can be accessed only from libobs.

**Comment by RytoEX** [comment]
This looks sane. @Lain-B ?

**Comment by RytoEX** [comment]
> > It is not used for the async delay filter?
> 
> The async delay filter has the same variable name but it is declared inside the filter.

Yeah, I realized this after a second look.

**Comment by RytoEX** [comment]
Actually, _can_ we even remove the member from the struct safely?

**Comment by tt2468** [comment]
> Actually, _can_ we even remove the member from the struct safely?

Yes, since the struct definition is private.

File: libobs/obs-internal.h [hunk 1]
```diff
@@ -756,7 +756,6 @@ struct obs_source {
 	volatile bool timing_set;
 	volatile uint64_t timing_adjust;
 	uint64_t resample_offset;
-	uint64_t last_audio_ts;
```
Comment (RytoEX) [inline libobs/obs-internal.h:759]: Can we even safely remove this from the struct definition?

Comment (Lain-B) [reply libobs/obs-internal.h:759]: Yep

**Review by Lain-B — APPROVED** [review]

**Pull request merged**